### PR TITLE
Install libwebp from binary release

### PR DESCRIPTION
**Pull request by szakarias**

### CODE REVIEW

File: tool/setup-webp.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    mkdir -p ${TARGET_DIRECTORY}
+    if [ "$?" != "0" ]; then
+        exit -1
+    fi
+}
+
+function download_and_extract() {
+    curl --retry 8 -L ${LIBWEBP_TAR_URI} | \
+      tar -xzv --strip-components=2 -C ${TARGET_DIRECTORY} --\
+      libwebp-${LIBWEBP_VERSION}-linux-x86-64/bin
```
Comment (jonasfj) [inline tool/setup-webp.sh:25]:
```suggestion
    curl --retry 8 -L "${LIBWEBP_TAR_URI}" | \
      tar -xzv --strip-components=2 -C "${TARGET_DIRECTORY}" --\
      "libwebp-${LIBWEBP_VERSION}-linux-x86-64/bin"
```

File: tool/setup-webp.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+}
+# Entry
+ensure_target_directory
+download_and_extract
```
Comment (jonasfj) [inline tool/setup-webp.sh:30]: Let's add a test that checks that `webp` and `gif2webp` are present and executable in the `TARGET_DIRECTORY`.

If not, print an error and exit non zero.

**Review by jonasfj — APPROVED** [review]

File: tool/setup-webp.sh (hunk 2 above)
Comment (jonasfj) [reply tool/setup-webp.sh:30]: that way, if in the future the contents of the tar.gz changes it we'll get an error.

File: tool/setup-webp.sh (hunk 1 above)
Comment (szakarias) [reply tool/setup-webp.sh:25]: done

File: tool/setup-webp.sh (hunk 2 above)
Comment (szakarias) [reply tool/setup-webp.sh:30]: done

**Pull request merged**